src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/LightCompiler.cs

```diff
@@ -1509,14 +1504,14 @@ private void CompileSwitchExpression(Expression expr)
                     return;
                 }
 
-                TypeCode switchType = System.Dynamic.Utils.TypeExtensions.GetTypeCode(node.SwitchValue.Type);
+                TypeCode switchType = node.SwitchValue.Type.GetTypeCode();
```

Comment:
What's our plan for when this lib gets upgraded to target netstandard1.7 and Type.GetTypeCode is available?  Change the extension method to use it?  I ask because it's a static, so if we were going to change all of the call sites to eliminate the extension method completely, it maps more closely to the previous form than it does to this one.